lib/rest-adapter.js

```diff
@@ -105,6 +105,27 @@ RestAdapter.prototype.connect = function(url) {
   this.connection = url;
 };
 
+/**
+ * Get the auth from the remotes or from the
+ * loopback options if available
+ * @param remotes
+ * @param args
+ */
+RestAdapter.prototype.extractAuth = function(remotes, args) {
+  var auth = remotes.auth;
+  // check for the loopback options to pass along.
+  // It may have the access token that can be used
+  if (!auth && args.length) {
+    var options = args[args.length - 1];
```

Comment:
This assumes that `args` do not contain the `callback` as the last argument, can we rely on this? I don't remember how exactly remote invocation works.